packages/sdk-android/src/kotlinParser.ts

```diff
@@ -173,13 +173,14 @@ export const parseMainActivitySync = (c: any) => {
 export const parseSplashActivitySync = (c: Context) => {
     const appFolder = getAppFolder(c);
     const { platform } = c;
-    const splashPath = 'app/src/main/java/rnv/SplashActivity.kt';
+    const splashTemplatePath = 'app/src/main/java/rnv_template/SplashActivity.java.tpl';
+    const splashPath = 'app/src/main/java/rnv/SplashActivity.java';
```

Comment:
@kasinskas the destination path needs to be inside the correct folder, like `src/main/java/com/app/....`, check https://github.com/flexn-io/renative/blob/c98b67d7a31f24526135ff42746f11eb47184dd8/packages/sdk-android/src/kotlinParser.ts#L127

Reply:
updated